test/api/adbc/test_adbc.cpp

```diff
@@ -1276,31 +1278,39 @@ TEST_CASE("Test AdbcConnectionGetObjects", "[adbc]") {
             })";
 		string expected_result_2 = R"({
                 'db_schema_name': main,
+                'db_schema_tables': NULL
+            })";
+		string expected_result_3 = R"({
+                'db_schema_name': main,
                 'db_schema_tables': [
                     {
                         'table_name': my_table,
                         'table_type': BASE TABLE,
                         'table_columns': [],
                         'table_constraints': []
+                    },
+                    {
+                        'table_name': my_view,
+                        'table_type': VIEW,
+                        'table_columns': [],
+                        'table_constraints': []
                     }
                 ]
-            })";
-		string expected_result_3 = R"({
-                'db_schema_name': pg_catalog,
-                'db_schema_tables': NULL
             })";
 		string expected_1_clean = StringUtil::Replace(
 		    StringUtil::Replace(StringUtil::Replace(expected_result_1, "\n", ""), "\t", ""), " ", "");
 		string expected_2_clean = StringUtil::Replace(
 		    StringUtil::Replace(StringUtil::Replace(expected_result_2, "\n", ""), "\t", ""), " ", "");
 		string expected_3_clean = StringUtil::Replace(
 		    StringUtil::Replace(StringUtil::Replace(expected_result_3, "\n", ""), "\t", ""), " ", "");
-		REQUIRE((StringUtil::Replace(res->GetValue(1, 2).ToString(), " ", "").find(expected_1_clean) >= 0));
-		REQUIRE((StringUtil::Replace(res->GetValue(1, 2).ToString(), " ", "").find(expected_2_clean) >= 0));
-		REQUIRE((StringUtil::Replace(res->GetValue(1, 2).ToString(), " ", "").find(expected_2_clean) >= 0));
+		REQUIRE((StringUtil::Replace(res->GetValue(1, 0).ToString(), " ", "").find(expected_1_clean) != string::npos));
+		REQUIRE((StringUtil::Replace(res->GetValue(1, 1).ToString(), " ", "").find(expected_2_clean) != string::npos));
+		REQUIRE((StringUtil::Replace(res->GetValue(1, 2).ToString(), " ", "").find(expected_3_clean) != string::npos));
```

Comment:
This is not related to `table_type` support.

Actual values should be `(1, 0)`, `(1, 1)`, `(1, 2)` not `(1, 2)`, `(1, 2)`, `(1, 2)`.

We can't detect "not found" by `find() >= 0`. We need to use `find() != string::npos` because `string::npos` is a large number not a negative number.